src/compose/service.ts

```diff
@@ -493,7 +493,7 @@ export class Service {
 			extraHosts: container.HostConfig.ExtraHosts || [],
 			ulimits,
 			stopSignal: (container.Config as any).StopSignal || '',
```

Comment:
@CameronDiver I think as @imrehg mentioned, we need to also make sure the default stopSignal == SIGTERM otherwise things don't work as expected.